test/TestImportTasks.py

```diff
@@ -0,0 +1,73 @@
+"""Calling 'import_task' from multiple files in different directories."""
+import os
+import shutil
+import tempfile
+import textwrap
+
+import pytest
+
+from ansiblelint.runner import Runner
+
+LAYOUT = {
+    'directories': ['tasks', 'tasks/subtasks'],
+    'files': {
+        'main.yml': textwrap.dedent(
+            """\
+            ---
+            - hosts: target
+              gather_facts: false
+              tasks:
+                - name: from main import task 1
+                  import_tasks: tasks/task_1.yml
+            """
+        ),
+        'tasks/task_1.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from task 1 import task 2
+              import_tasks: tasks/task_2.yml
+            """
+        ),
+        'tasks/task_2.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from task 2 import subtask 1
+              import_tasks: tasks/subtasks/subtask_1.yml
+            """
+        ),
+        'tasks/subtasks/subtask_1.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from subtask 1 import subtask 2
+              import_tasks: tasks/subtasks/subtask_2.yml
+            """
+        ),
+        'tasks/subtasks/subtask_2.yml': textwrap.dedent(
+            """\
+            ---
+            - name: from subtask 2 do something
+              debug:
+                msg: |
+                  Something...
+            """
+        ),
+    },
+}
+
+
+@pytest.mark.xfail
+def test_import_tasks(default_rules_collection):
+    """Calling 'import_task' from multiple files in different directories."""
```

Comment:
What does it check?

Reply:
Changed. Now it includes more descriptive definition. We are checking the file path evaluation because the use of import_tasks or include_tasks in different directories leads to an incorrectly evaluated file path, which is used by import_tasks or include_tasks.